index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+     <title> Portfolio</title>
+     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+</head>
+
+<body>
+
+<header>
+
+    <ul class="container1">
+    <li class="logo">Welcome</li>
+    <li class="item-center"> </li>
+    <li><i class="fa fa-bars"></i></li>
+    </ul>
+
+</header>
+<div class="background">
+<section class="container2">
+
+    <h1 class="greeting">Hey there. I'm Houssam I am a software developer </h1>
+    <p class="presentation"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+don’t hestiate to contact me.</p>
+
+ </section>
+ <footer>
+    <div>LET'S CONNECT</div>
+        <img alt="icon" src="icons.png"></footer>
```

Comment:
The social icons are a list of icons:

- [ ] Please use a `ul` tag to wrap all social icons 🤔 something like `<ul><li><li></ul>`

- [ ] Please use an `li` tag per icon and an `a` tag for the icon link 🤔 ... something like this  `<li><a></a></li>`